src/sqlparse.yrl

```diff
@@ -1084,6 +1085,10 @@ any_all_some -> SOME : some.
 
 existence_test -> EXISTS subquery : {exists, '$2'}.
 
+table_coll_expr -> TABLE '(' column_ref   ')' : {table_coll_expr, '$3'}.
+table_coll_expr -> TABLE '(' function_ref ')' : {table_coll_expr, '$3'}.
+table_coll_expr -> TABLE '(' subquery     ')' : {table_coll_expr, '$3'}.

```

Comment:
Missing implementation (and tests) for the `(+)` part of the following rule:
![image](https://user-images.githubusercontent.com/913008/39349719-3d8e004a-49fc-11e8-878b-1ef355281c4a.png)